hack/update-bazel.sh

```diff
@@ -17,8 +17,9 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-export KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+export
```

Comment:
Is this `export` on its own intentional?

Reply:
yes! fun shell quirk: if you do `local foo=$(compute_foo)` or `export FOO=$(compute_foo)` with `errexit`, you won't actually exit if `compute_foo` fails. This is because bash errors are line-oriented, and `export` / `local` run last and always exit with success.

Reply:
🤦‍♂️ this should be `export KUBE_ROOT` and I should not reply to code review on my phone 🙃, will fix in a minute 😅

Reply:
fixed, thank you.

Reply:
If it makes you feel better, I did the review from my phone😁😇😂